server/insights/src/test/java/org/infinispan/server/insights/InsightsActivationTest.java

```diff
@@ -11,35 +15,46 @@
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.Test;
 
+import com.redhat.insights.config.EnvAndSysPropsInsightsConfiguration;
 import com.redhat.insights.reports.InsightsReport;
 
 @Test(groups = "functional", testName = "server.insights.InsightsActivationTest")
 public class InsightsActivationTest extends AbstractInfinispanTest {
 
-   public void test() throws Exception {
-      System.setProperty(InsightsModule.REDHAT_INSIGHTS_ACTIVATION_PROPERTY_NAME, InsightsActivation.LOCAL.name());
+   public void enabled() throws Exception {
+      Path tmpDir = Paths.get(System.getProperty("java.io.tmpdir"), "blablabla");
+      Files.createDirectories(tmpDir);
+      System.setProperty(InsightsModule.REDHAT_INSIGHTS_ACTIVATION_PROPERTY_NAME, InsightsActivation.ENABLED.name());
+      System.setProperty(InfinispanInsightsConfiguration.ENV_RHEL_MACHINE_ID_FILE_PATH.toLowerCase().replace('_', '.'), tmpDir.toString());
```

Comment:
Here we seem to set a directory as a target for something that should be a file. Probably enough to get far enough to generate a file report even though I'd expect Insights client to read the file content (the machine id) and use it somehow in the report. Maybe it just serves as a check that the insight-client rpm is installed

Reply:
We could have created a non-directory file instead of a directory and assigned it to the `ENV_RHEL_MACHINE_ID_FILE_PATH`. For simplicity, since we had to create a directory to be assigned to the `ENV_ARCHIVE_UPLOAD_DIR`, I thought that it was safe to use the same path.
If you instead think that it is necessary to use a proper file here, I can change it. For me it is the same.